lib/SQL/Translator/Parser/Oracle.pm

```diff
@@ -489,7 +489,17 @@ parens_name_list : '(' NAME(s /,/) ')'
 field_meta : default_val
     | column_constraint
 
-default_val  : /default/i VALUE
+default_val  : 
+    /default/i CURRENT_TIMESTAMP
```

Comment:
just to get this straight: this is special casing CURRENT_TIMESTAMP b/c it's a magic constant in the Oracle grammar, and you don't want SQLT to quote it down the line?

If that's the case, I don't know if this is the right place to take care of special casing it here. If you look at the SQLite parser, they special case it in the rule for `VALUE`, which makes more sense to me.

Reply:
ah, looking at it again, I see that's exactly what the MySQL parser does. So then I'm okay w/ this, though I think the SQLite way makes more sense

Reply:
I handled it that way because of how it was handled for MySQL and CURRENT_TIMESTAMP is handled the same way by Oracle and MySQL databases. This is my first time looking at the parser syntax and its quite confusing so I tried to keep the changes as simple and as close to what I could understand as possible.